### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by zulkernine**
Successfully installed and tracking command executed

### CODE REVIEW

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/utils/io.py  1
- wplay/wchat.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5184811&bid=17012055)

**Comment by zulkernine** [comment]
> Codacy Here is an overview of what got changed by this pull request:
> 
> ```diff
> Complexity increasing per file
> ==============================
> - wplay/utils/io.py  1
> - wplay/wchat.py  2
>          
> ```
> 
> See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5184811&bid=17012055)

What should I do now?

**Comment by zulkernine** [comment]
> @zulkernine Add tracking command execution screenshot and solve only one issue from one PR. :)

How can I now place these in two different issues?
Should I delete change in wchat and io and wait until first PR (screenshot) get resolved?

**Comment by zulkernine** [comment]
Should I close this pull request then ??
And make new pull request for that screenshot only ??

**Comment by zulkernine** [comment]
@rpotter12 should I close it and make new PR regarding that initial issue (screen shot of execution of tracking command) ??

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5184811&bid=17012055)

File: wplay/utils/io.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def ask_user_for_message():
 def ask_user_for_message_breakline_mode():
     message = []
     i = 0
-    print("Write your message ('Enter' to breakline)('.' alone to finish):")
+    print("Write your message ('Enter' to breakline)('.' alone to finish):")
```
Comment (rpotter12) [inline wplay/utils/io.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5184811)

**Comment by zulkernine** [comment]
> @zulkernine Add tracking command execution screenshot and solve only one issue from one PR. :)

Done.

**Comment by zulkernine** [comment]
Please review the changes and merge it .

**Comment by priyansh19** [comment]
Still, you haven't added ScreenShots in the Comment Section of this PR.
Do Add them ..then you are good to go :)

**Comment by zulkernine** [comment]
> Still, you haven't added ScreenShots in the Comment Section of this PR.
> Do Add them ..then you are good to go :)

Here's the screen shot 😅😅
zulkernine_executingTrackingCommand

**Comment by zulkernine** [comment]
It it alright now?

**Comment by zulkernine** [comment]
Is this screenshot alright?

**Pull request merged**